tor/controller.go

```diff
@@ -168,26 +189,73 @@ func parseTorReply(reply string) map[string]string {
 }
 
 // authenticate authenticates the connection between the controller and the
-// Tor server using the SAFECOOKIE or NULL authentication method.
+// Tor server using the SAFECOOKIE authentication method.
```

Comment:
This comment is no longer accurate, this function now probes all auth methods that are supported.